packages/wonder-blocks-core/components/no-ssr.test.js

```diff
@@ -0,0 +1,70 @@
+// @flow
+import * as React from "react";
+// import * as ReactDOM from "react-dom";
+import * as ReactDOMServer from "react-dom/server";
+
+import {mount} from "enzyme";
+
+import NoSSR from "./no-ssr.js";
+
+describe("NoSSR", () => {
+    test("renders a placeholder first, then the actual content", (done) => {
```

Comment:
How does this test assert the order?

Reply:
Because the  assert is called when the inside renders, and the assert checks that the placeholder was already called. Therefore, the placeholder HAD to have been called first.

Reply:
Ah... I got it.  I was confused b/c I mistook `assert` for a general `assert` function that might of come from node's `assert` package.  Time to go for a walk and get some fresh air.